experiments/compositions/composition/api/v1alpha1/expanderversion_types.go

```diff
@@ -38,12 +38,12 @@ const (
 
 // ExpanderVersionSpec defines the desired state of ExpanderVersion
 type ExpanderVersionSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
-
 	// ImageRegistry is the designated registry for where to pull the named expander image
 	ImageRegistry string `json:"imageRegistry,omitempty"`
 
+	// Image if different from removePrefix(expanderversion.name , "composition-")
+	Image string `json:"image,omitempty"`
```

Comment:
for my own clarification, do we have an example for of this image string, is it going to be version?

Reply:
Guessing this will look like "gcr.io/MY-PROJECT/MY-EXPANDER:v1.0.12" or some such

Reply:
Adding optional image that allows user to specify an image instead of us deducing image name from the expander version name. 

```
  imageRegistry: gcr.io/krmapihosting-release
  image: expander-jinja2
  validVersions:
     ...
  ```